packages/terra-site/src/App.jsx

```diff
@@ -189,6 +189,7 @@ class App extends React.Component {
           <List.Item content={<Link onClick={this.handleResetScroll} to="/site/table">Table</Link>} />
           <List.Item content={<TableComponentNavigation onClick={this.handleResetScroll} />} />
           <List.Item content={<Link onClick={this.handleResetScroll} to="/site/tabs">Tabs</Link>} />
+          <List.Item content={<Link onClick={this.handleResetScroll} to="/site/Tag">Tag</Link>} />
```

Comment:
nit: In the `to` prop, use lowercase `t` for the word `tag`. It matches casing for other documentation links.

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/a6617b0978ac0a4b8bfd27316968c1543d5be2df).